src/index.ts

```diff
@@ -41,6 +41,26 @@ const parseNoLookup = (data: ArrayBuffer): Array<GRIBPacketValues> => {
   return packets
 }
 
-export const GRIB = { parse, parseNoLookup }
+/**
+ * Lookup data point from GRIB packet based on lat/lon coordinates.
+ *
+ * @param packet GRIB packet
+ * @param lat Latitude
+ * @param lon Longitude
+ */
+const lookupDataPoint = (packet: GRIBPacket, lat: number, lon: number): number | null => {
```

Comment:
Could you explain a bit what this function would be used for?

Reply:
My intent was to have a way to get data associated with certain lat, lon from the packet. I wasn't able to find anything for this that's currently in the library.

What's your view on this?

Reply:
I think it's definitely something that is needed, however, it looks like this would only be compatible with GRIBs that utilize [Grid Def Template 3.0](https://www.nco.ncep.noaa.gov/pmb/docs/grib2/grib2_doc/grib2_temp3-0.shtml).

For compatibility, I feel like this function should first check that the packet utilizes a compatible Grid Template.